app/models/pglogical_subscription.rb

```diff
@@ -86,16 +86,24 @@ def pglogical(refresh = false)
     self.class.pglogical(refresh)
   end
 
-  def self.validate(host, port, user, password, dbname, id)
+  def self.validate(host: nil, port: nil, user: nil, password: nil, dbname: nil, id: nil)
```

Comment:
Looking back I'm actually surprised that worked. @Fryguy did I do that right or is something unexpected happening?

I thought optional arguments had to be at the end of the list of method args or did I make that up?